### DISCUSSION TITLE
Change each Unkown Album over WEB GUI?

### DISCUSSION THREAD

**Discussion by jlmlr** [blackcandy-org/blackcandy#385] [General]
All Works fine. Hostet over docker. But i found the function 
db/migrate/20240103072942_add_default_name_to_unknown_albums.rb

Is there a way to change tag for each "Unkown Album" over web access? 

![grafik](https://github.com/blackcandy-org/blackcandy/assets/111344026/21a8ce2d-4c1a-433e-8ab3-e488d44aa9f5)

**Comment by aidewoode** [comment]
Right now there is no way you can change album title through web UI. But you can using some other tools like Kid3 to edit the title tag of your audio files. If you have already enabled the media listener in settings page then the album info will be updated automatically.

**Reply by jlmlr** [reply]
Works fine! Thank you.
